language-design.md

```diff
@@ -106,7 +209,25 @@ you do so? If not, why not?
 
 **Response**
 
+The quote that we already cited above is very relevant:
+
+> If your standard is natural language, division/fraction notation isn't particularly intuitive.
+> But to someone who knows the notation, ab / b is very intuitive, and it's immediately obvious that
+> you can cancel the bs. So the goal of an intuitive programming syntax shouldn't be to emulate
+> natural language, necessarily, but to provide intuitive hooks and syntax for the types of things
+> that programmers do a lot. 
+> [Comment by "Peter" ([Source](https://www.fastcodesign.com/1665735/why-arent-computer-programming-languages-designed-better#comment-bc08f700-3c06-11e1-921b-9794fc737ae8))] [Pavlus, 2012]
+
+There are (at least) two main considerations for syntax. The first is how well it fits the task at hand, *in a vacuum*. That is, how good it is considering only the language and the task, and not what users are already used to or know. The other is how easy it will be for real users to adopt. Considering this, the analogy with mathematics is apt. After all, mathematicians do *not* use natural language. They use mathematical notation, because
+it is simply better for the task. However, when new ideas come along, to begin with, the ideas are explained using natural language and older notation. The optimal descriptions for learning concepts are not the same as the optimal descriptions for using them.
```

Comment:
Good point. Natural language does a great job at explaining, but not necessarily in the language itself.